snapcraft/_store.py

```diff
@@ -433,8 +437,9 @@ def list_registered():
 
 
 def _get_usable_keys(name=None):
+    snap_path = get_host_tool_path(command_name="snap", package_name="snapd")
```

Comment:
What if there's no `snap`? Shouldn't this raise `storeapi.errors.MissingSnapdError`? How that case is managed?

If not, shouldn't that exception be removed from `storeapi.errors`?

Reply:
get_host_tool_path raises an appropriately formatted exception which is generic enough to be valid on any system Snapcraft is ever ported to https://github.com/snapcore/snapcraft/blob/e64c08aeddbc98df01c8644ce1d2762af2c30620/snapcraft/file_utils.py#L340

I will check though if this MissingSnapdError needs to be cleaned up